frontend/src/metabase/public/lib/types.ts

```diff
@@ -15,34 +15,41 @@ export type EmbedResourceParameter = {
   type: string;
 };
 
-export type EmbeddingParameters = Record<string, string>;
```

Comment:
We were misusing this type for both embedding parameters ("disabled" | "enabled" | "locked") and parameter values e.g. "Doohikey"

Reply:
My 2c here: it's really unclear how `EmbeddingParametersOptions` different from `EmbeddingParametersSettings`. Naming is hard, but maybe there's a better way? 

WDYT if we keep the name `EmbeddingParameters` for what is coming in the `embedding_params` key and rename `EmbeddingParametersOptions` into something like `EmbeddingParameterVisibility`?

Reply:
I like `EmbeddingParameterVisibility`.

And `EmbeddingParameters` could also work since it's from `embedding_params` like you said.

I agree with what you've suggested here.